packages/divoc/README.md

```diff
@@ -0,0 +1,31 @@
+# language-divoc <img src='./assets/square.png' width="30" height="30"/>
+
+An OpenFn **_adaptor_** for building integration jobs for use with the divoc API.
+
+## Documentation
+
+View the [docs site](https://docs.openfn.org/adaptors/packages/divoc-docs)
+for full technical documentation.
+
+### Configuration
+
+View the
+[configuration-schema](https://docs.openfn.org/adaptors/packages/divoc-configuration-schema/)
+for required and optional `configuration` properties.
+
+### A example expression using the `create` function
+
+```js
+post('patient', { name: 'Bukayo' });
```

Comment:
needs updating!

Reply:
Mutchi is voting to get rid of examples in the readme entirely, and I think he might be right. What is the value of an example like this? It'll just drift out of date, it's not a full tutorial, and better maintained examples can be found in the adaptor docs.

Reply:
happy with that. drop it!

Reply:
I'll commit a template change straight to main. @PiusKariuki Can you please just remove the example here? Thanks